docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md

```diff
@@ -11,25 +11,35 @@ import TabItem from '@theme/TabItem';
 
 <img src={useBaseUrl('img/integrations/containers-orchestration/rabbitmq.png')} alt="icon" width="45"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
 
-The [RabbitMQ](https://www.rabbitmq.com/getstarted.html) app is a unified log app. Preconfigured dashboards provide insight into error logs. RabbitMQ logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+The Sumo Logic app for [RabbitMQ](https://www.rabbitmq.com/getstarted.html) is a unified log solution that includes preconfigured dashboards offering insights into error logs and metrics. RabbitMQ logs and metrics are transmitted to Sumo Logic via the OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
```

Comment:
this is incorrect. Logs are through filelog receiver and metric will be through rabbit mq receiver

Reply:
fixed it.